src/documentation.ts

```diff
@@ -25,6 +23,8 @@ import {
   ensureShortDescription,
   reformatParamsInPath,
 } from "./documentation-helpers";
+import { mapLogicalContainer } from "./logical-container";
+import { Method } from "./method";
```

Comment:
I'm not sure what's causing the change to the imports arrangement in several files of this PR, but if it's possible, please reduce the `diff` by restoring the original order of import (except it's not causing ESLint errors)

Reply:
It's because I had `"source.organizeImports": true` set for my typescript files in vscode.

Sorry for the confusion.